static/js/src/advantage/manage-subscription.js

```diff
@@ -0,0 +1,283 @@
+        <p>If you cancel this subscription:</p>
+        <ul>
+          <li>No additional charge will be incurred.</li>
+          <li>The ${
+            machineCount > 1
+              ? `<strong>${machineCount} machines</strong>`
+              : "<strong>machine</strong>"
+          } will stop receiving updates and services at the end of the billing period.</li>
+          </ul>
+        <p>Want help or advice? <a href="/contact-us">Chat with us</a>.</p>
+        <div id="cancel-modal-field-wrapper" class="u-align--right">
+          <label for="confirm-cancel">Please type <strong>cancel</strong> to confirm.</label>
+        </div>
+        <div id="cancel-modal-buttons-wrapper" class="u-align--right">
+        </div>
+      </div>
+    </div>
+  `;
+  container.innerHTML = content;
+  document.body.appendChild(container);
+
+  const fieldWrapper = document.querySelector("#cancel-modal-field-wrapper");
+  fieldWrapper.appendChild(confirmCancelField);
+
+  const buttonWrapper = document.querySelector("#cancel-modal-buttons-wrapper");
+  buttonWrapper.appendChild(goBackButton);
+  buttonWrapper.appendChild(confirmCancelButton);
+}
+
+function handleCancelChangesClick(id, VPSize) {
+  // Hide edit options
+  const previewSection = document.querySelector(
+    `#view-mode--${id}[data-viewport="${VPSize}"]`
+  );
+  const editSection = document.querySelector(
+    `#edit-mode--${id}[data-viewport="${VPSize}"]`
+  );
+  previewSection.classList.remove("u-hide");
+  editSection.classList.add("u-hide");
+
+  // Reset input and help text
+  const resizeField = document.querySelector(
+    `#resize-input--${id}[data-viewport="${VPSize}"]`
+  );
+  resizeField.oninput = () => {};
+  resizeField.value = resizeField.defaultValue;
+  const resizeSummary = document.querySelector(
+    `#resize-summary--${id}[data-viewport="${VPSize}"]`
+  );
+  const newPayment = document.querySelector(
+    `#new-payment--${id}[data-viewport="${VPSize}"]`
+  );
+  const updateButton = document.querySelector(
+    `#save-changes--${id}[data-viewport="${VPSize}"]`
+  );
+  resizeSummary.classList.add("u-hide");
+  newPayment.classList.add("u-hide");
+  updateButton.disabled = true;
+
+  // Remove buttons handlers
+  const cancelSubscriptionButton = document.querySelector(
+    `#cancel-subscription--${id}[data-viewport="${VPSize}"]`
+  );
+  const cancelChangesButton = document.querySelector(
+    `#cancel-changes--${id}[data-viewport="${VPSize}"]`
+  );
+  updateButton.onclick = () => {};
+  cancelChangesButton.onclick = () => {};
+  if (cancelSubscriptionButton) cancelSubscriptionButton.onclick = () => {};
+}
+
+function handleChange(e, id, VPSize) {
+  const defaultValue = Number.parseInt(e.target.defaultValue);
+  let newValue = Number.parseInt(e.target.value);
+
+  // const unitPrice = Number.parseFloat(e.target.dataset.unitPrice.split(" ")[0]);
+  // const nextPayment = Number.parseFloat(
+  //   e.target.dataset.nextPayment.split(" ")[0]
+  // );
```

Comment:
Can you remove the commented out code please

Reply:
Whoops! Of course!